modin/_compat/pandas_api/latest/base.py

```diff
@@ -241,7 +241,7 @@ def rolling(
             closed=closed,
             method=method,
         )
-
+        
```

Comment:
Nit: Can we remove this change unless it is needed by the linter?

Reply:
Done.